ChegeBryan/politico
# #165906352 Fetch API user login

#### What does this PR do?

Builds the log in for the app ftom the frontend.

#### Description of Task to be completed?

- [x] Create the login logic the consumes the app API 

#### How should this be manually tested?
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Checkout the branch `ft-fetch-api-login-165906352` open the `index.html` page to serve the homepage.  Click on the sign in option use the admin logins as folows:
```json
{
    "email": "admin@politico.org",
    "password": "2019NBO37"
}
```
#### What are the relevant pivotal tracker stories?
[#165906352](https://www.pivotaltracker.com/story/show/165906352)
